intro-Globus/ch_Access.tex

```diff
@@ -0,0 +1,32 @@
+\chapter{Access to the VSC Cloud}
+
+The Globus platform enables developers to provide robust file transfer, sharing and search capabilities within their own research data applications and services, while leveraging advanced identity management, single sign-on, and authorization capabilities.
+
+\section{Log in with an existing identity}\label{login}
+
+Visit \href{https://www.globus.org}{www.globus.org} and click \emph{Login} at the top of the page. On the Globus login page, choose an organization you are already registered with, such as your school or your employer.
+
+\begin{center}
+\includegraphics[width=0.5\textwidth]{img/access-login-screen.png}
+\end{center}
+
+When you find it, click \emph{Continue}. If you cannot find your organization in the list, you can use Google, ORCID, or Globus ID, all three of which allow you to create new accounts if you do not already have one.
```

Comment:
this guide is for vsc user, if they do not find their origanoisation, they should contact support `data@vscentrum.be`